src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -106,6 +109,16 @@ const ActivityThread = ({
                     translations={translations}
                 />
             )}
+
+            {onReplyCreate ? (
```

Comment:
It looks like `onReplyCreate` will always be defined due to the default `noop` value being assigned above during destructuring. Is `onReplyCreate` nullable? If so, its type doesn't indicate that.

Reply:
True. That was a mistake on my side. `onReplyCreate` should be optional on `ActivityTHread` but not on `ActivityThreadReplyForm`.
I'll add one more test to catch that we don't show reply button when `onReplyCreate` isn't passed.